src/rect.h

```diff
@@ -0,0 +1,159 @@
+#ifndef RAYTRACING_RECT_H
+#define RAYTRACING_RECT_H
+
+
+#include "hitable.h"
+#include "material.h"
+
+
+class XY_Rect : public Hitable
```

Comment:
@Ale32 needed to define my own `Rect` class, because with just a `Sphere` I couldn't test the instancing as is defined here - with a specific class for rotating the instance on a specific axis.

Reply:
Cool